### ISSUE TITLE
Apache CONFIGKEY not able to retrieved in Laravel Jobs

### ISSUE THREAD

**Issue by yasharrow** [mrgswift/laravel-encryptenv#6]
@mrgswift Thanks for this awesome plugin, appreciated 👏 

I have following concern if you could share some inputs for specific laravel jobs execution after encrypting env file,

$_SERVER is not getting apache configuration key while running laravel Jobs and its failing on first step on connecting database as it is not able to communicate with apache configuration while running cronjobs.

Any way to set $_SERVER for laravel jobs as it is running under the hood cron scripts.

Thanks.

**Comment by adamdenverco** [comment]
@mrgswift, thank you as well for this plugin. I have the same bug: works great from a browser, but CONFIGKEY is not available from the command line.

**Comment by adamdenverco** [comment]
The temporary solution I have is to include the CONFIG key in the command:

If running a single command:
`export CONFIGKEY='asdfasdf' && php artisan command:my_command`

If running via cron:
`* * * * * cd /your/laravel/dir && export CONFIGKEY='asdfasdf' && php artisan schedule:run >> /dev/null 2>&1`

Solution from here:
https://stackoverflow.com/a/18098714/2624129

**Comment by adamdenverco** [comment]
Follow up, my solution works for running a single command in command line, but not for Laravel scheduler.

I see scheduler outputs this:
`Running scheduled command: '/usr/bin/php' 'artisan' command:my_command >>  2>&1`

It looks like it's simulating running the manual running of command-line code but excludes this code:
` export CONFIGKEY='asdfasdf' && `

Investigating a solution.

**Comment by mrgswift** [comment]
When I made this package, the fact that the CONFIGKEY is not available to the system environment was by design.  However, I do see the practical need for this to work with laravel console commands.  I need to think about this more to figure out a secure way to grab the CONFIGKEY without exposing it to the console script.

**Comment by yasharrow** [comment]
> When I made this package, the fact that the CONFIGKEY is not available to the system environment was by design. However, I do see the practical need for this to work with laravel console commands. I need to think about this more to figure out a secure way to grab the CONFIGKEY without exposing it to the console script.

@mrgswift any workaround you can suggest to achieve job executions using encrypted credentials? I can work upon that and submit PR in support to that. Thanks.

**Comment by patake** [comment]
@mrgswift awesome job. 
Any progress with the enhancement to not expose the CONFIGKEY in console commands?
Thanks.

**Comment by yasharrow** [comment]
> @mrgswift awesome job. Any progress with the enhancement to not expose the CONFIGKEY in console commands? Thanks.

@patake There is no solution in code side to achieve this.

You can keep this key in supervisor config if you want your jobs to run and that will work.

**Comment by mrgswift** [comment]
@yasharrow @patake @adamdenverco
Starting with v1.0.8, you are now able to run console commands using your CONFIGKEY.  A new console command `artisan encryptenv:console` was added for you to run your own console commands.  I tested this with Laravel scheduler and commands like php artisan migrate.  These commands tested successfully and were able to query and write to the database when the database env variables are encrypted.  I updated the README with the new console command and how to use it.
